src/components/list/List.js

```diff
@@ -47,6 +47,20 @@ export const List = ({
     'data-list-column': true,
   })
 
+  const onCellMouseEnter = (index) => {
+      const cells = document.getElementsByClassName(`index-row-${index}`);
```

Comment:
There is rarely a need to query the DOM manually like this in React applications and this no exception... I think we can get rid of this imperative code by using something like this:

```javascript
const [currentHoveredIndex, setCurrentHoveredIndex] = React.useState(null)

{sortedByAccessor.map((data, index) =>
          Object.keys(data).map((key, i) => {
            const hoveredClassName = `cellhovered + ${rowClass}`
            const regularClassName = rowClass
            return (
              key !== 'id' && (
                <span
                  style={conditionalBorderRadius(i)}
                  onClick={() => handleRowClick && handleRowClick(data)}
                  key={uniqueId()}
                  className={
                    currentHoveredIndex === index
                      ? hoveredClassName
                      : regularClassName
                  }
                  onMouseEnter={() => setCurrentHoveredIndex(index)}
                  onMouseLeave={() => setCurrentHoveredIndex(null)}
                >
                  {isLoading
                    ? ''
                    : typeof data[key] === 'function'
                    ? data[key]()
                    : data[key]}
                </span>
              )
            )
          }),
```

What do you think?